tests/e2e/specs/shopper/active-filters.test.ts

```diff
@@ -113,18 +113,21 @@ describe( 'Shopper → Active Filters Block', () => {
 				}
 			);
 
+			await page.waitForSelector(
+				selectors.frontend.productsList + '.is-loading',
```

Comment:
Just curious, why did you use this method of concatenation over template literal?